### PR TITLE
Clean up v1

### PR DESCRIPTION
**Pull request by limsammy**
Related Issue: #48 

Summary: Styled item show page (barebones) and cleaned up seeding a bit. **RUN RAKE DB:RESET**

Author(s): sam

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -8,11 +10,12 @@
...
+  category = categories.shuffle.first
+
 	puts "created #{category.title}"
-	10.times do |num|
-		Item.create(title: "title#{num}", description: "descrip#{num}", price: num + rand(5), image: "http://lorempixel.com/400/200", category_id: category.id)
-	end
-	items = Item.pluck(:title)
-	puts "Created #{items}"
+
+	item = Item.create(title: Faker::Music.instrument, description: Faker::RickAndMorty.quote, price: num + rand(5), image: "http://lorempixel.com/400/200", category_id: category.id)
```
Comment (amhursh) [inline db/seeds.rb:18]: Much better. Get shwifty.

File: Gemfile [hunk 2]
```diff
@@ -41,6 +41,7 @@ group :development, :test do
   gem 'pry-rails'
   gem 'factory_girl_rails'
   gem 'orderly'
+  gem 'faker'
```
Comment (amhursh) [inline Gemfile:44]: excellent choice.

Comment (randy-darsh) [reply Gemfile:44]: YES!

Comment (randy-darsh) [reply Gemfile]: most good

File: Gemfile.lock [hunk 3]
```diff
@@ -229,4 +232,4 @@ DEPENDENCIES
   web-console (>= 3.3.0)
 
 BUNDLED WITH
-   1.15.4
+   1.16.0.pre.2
```
Comment (randy-darsh) [inline Gemfile.lock:235]: upgrading bundler ftw

File: app/views/items/index.html.erb [hunk 4]
```diff
@@ -8,7 +8,7 @@
 			<div class="item_card index_item_<%= item.id %>">
 				<img class="item-card-img" src="<%= item.image %>" alt="">
 				<h3><%= link_to item.title, item_path(item) %></h3>
-				<p><%= link_to "Add to Cart", cart_path(item_id: item.id), method: :post %></p>
+				<p><%= link_to "Add to Cart", cart_path(item_id: item.id), method: :post, class: "cart btn btn-outline-info" %></p>
```
Comment (randy-darsh) [inline app/views/items/index.html.erb]: i like it

File: db/seeds.rb [hunk 5]
```diff
@@ -8,11 +10,12 @@
 
 categories = Category.all
 
-categories.each do |category|
+20.times do |num|
+  category = categories.shuffle.first
+
 	puts "created #{category.title}"
-	10.times do |num|
-		Item.create(title: "title#{num}", description: "descrip#{num}", price: num + rand(5), image: "http://lorempixel.com/400/200", category_id: category.id)
```
Comment (randy-darsh) [inline db/seeds.rb:14]: indentation? AARON!?

File: db/seeds.rb (hunk 1 above)
Comment (randy-darsh) [reply db/seeds.rb:18]: i love shwiftyness. most good

**Pull request merged**